installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -140,16 +147,23 @@ expects: fedora|centos[/<version>], <ip address>, update, v[1-9].radia.run"
     if [[ $base == $host ]]; then
         host=$host.radia.run
     fi
-    vagrant_dev_private_net=${vagrant_dev_private_net-1}
-    # vbguest is pretty broken so we are defaulting to off for now
-    vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
+    if [[ ${vagrant_dev_vm_devbox:+1} ]]; then
+        vagrant_dev_no_mounts=${vagrant_dev_no_mounts-1}
+        vagrant_dev_no_nfs_src=${vagrant_dev_no_nfs_src-1}
+        vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
+        vagrant_dev_private_net=${vagrant_dev_private_net-}
+        vagrant_dev_provision_eth1=${vagrant_dev_provision_eth1-}
```

Comment:
`vagrant_dev_provision_eth1=${vagrant_dev_provision_eth1-1}`

Reply:
Why? We aren't handling the the rsconf case right now so I don't think eth1 needs to be properly setup. If we are going to turn it on then we are going to have to figure out ip addresses for these machines.

Reply:
Ah, didn't realize. Ignore.